pom.xml

```diff
@@ -19,7 +19,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <abstract-operator.version>0.6.14</abstract-operator.version>
+        <abstract-operator.version>0.6.16-SNAPSHOT</abstract-operator.version>
```

Comment:
this should be a non-snapshot version once the corresponding abstract operator is released. Just a note so that we will not merge it as it is, but CI should fail on this anyway

Reply:
fixed